pages/01.Home/02.Contributing/01.Style Guide/docs.en.md

```diff
@@ -0,0 +1,289 @@
+
+<details><summary>A markdown syntax</summary>
+
+##### `<codeblock>` markdown syntax
+
+```markdown
+
+/**
+ * Constructor.
+ *
+ * @param string $environment The environment
+ * @param bool   $debug       Whether to enable debugging or not
+ *
+ * @api
+ */
+public function __construct($environment, $debug)
+{
+    defined('MAUTIC_ENV') or define('MAUTIC_ENV', $environment);
+    defined('MAUTIC_VERSION') or define(
+        'MAUTIC_VERSION',
+        self::MAJOR_VERSION.'.'.self::MINOR_VERSION.'.'.self::PATCH_VERSION.self::EXTRA_VERSION
+    );
+
+    parent::__construct($environment, $debug);
+}
+
+```
+
+</details>
+</details>
+
+<details><summary>example nested codeblock</summary>
+
+##### Example nested `<codeblock>`
+
+1. START list item
+
+   Example nested `<codeblock>`
+
+    ```php
+    /**
+     * Constructor.
+     *
+     * @param string $environment The environment
+     * @param bool   $debug       Whether to enable debugging or not
+     *
+     * @api
+     */
+    public function __construct($environment, $debug)
+    {
+        defined('MAUTIC_ENV') or define('MAUTIC_ENV', $environment);
+        defined('MAUTIC_VERSION') or define(
+            'MAUTIC_VERSION',
+            self::MAJOR_VERSION.'.'.self::MINOR_VERSION.'.'.self::PATCH_VERSION.self::EXTRA_VERSION
+        );
+
+        parent::__construct($environment, $debug);
+    }
+    ```
+
+1. END list item
+
+<details><summary>markdown syntax</summary>
+
+##### Nested `<codeblock>` markdown syntax
+
+```markdown
+    ```php
+    /**
+     * Constructor.
+     *
+     * @param string $environment The environment
+     * @param bool   $debug       Whether to enable debugging or not
+     *
+     * @api
+     */
+    public function __construct($environment, $debug)
+    {
+        defined('MAUTIC_ENV') or define('MAUTIC_ENV', $environment);
+        defined('MAUTIC_VERSION') or define(
+            'MAUTIC_VERSION',
+            self::MAJOR_VERSION.'.'.self::MINOR_VERSION.'.'.self::PATCH_VERSION.self::EXTRA_VERSION
+        );
+
+        parent::__construct($environment, $debug);
+    }
+    ```
+```
+
+</details>
+</details>
+
+### Links
+
+[link macro]: <http://example.com>
+[testing]: <./../plugins/integration_test.html>
+
+Often you'll want to make a link to another place in the documentation.  We prefer to group links at the bottom of a page, and provide a reference macro in the text.
+This make linking to the same place much easier.  In Markdown, it looks like this:
+
+#### Absolute links
+
+You can have different links in your text, some leading [here][link macro] and others leading [there][link macro].
+
+```markdown
+You can have different links in your text, some leading [here][link macro] and others leading [there][link macro].
+
+[link macro]: <http://example.com>
+```
+
+#### Relative links
+
+These will link to [`plugins/integration_test.html`][testing] on the documentation website created from the [_`plugins/integration_test.md`_][testing] source file.
```

Comment:
This uses the old format, let's update it to avoid confusion!

```suggestion
These will link to [`/plugins/plugin-resources/testing-integrations`][testing] on the documentation website created from the [_`pages/13.Plugins/00.General%20Resources/01.Integration%20test/docs.en.md`_][testing] source file.  These should still use a link macro as with absolute links.
```